docs/pages/debugging/devtools-plugins.mdx

```diff
@@ -0,0 +1,195 @@
+
+It is recommended that you wrap this code in a conditional that only runs in development mode so that it is not included in the production build of your app:
+
+```jsx
+if (__DEV__) {
+  // initialize your DevTools plugin here
+}
+```
+
+In some cases, you may need to interact with a plugin directly. All plugins communicate through exports from `expo/devtools`, and you can send and listen to messages through `connectPluginFromDevToolsAsync`:
+
+```jsx
+import { connectPluginFromDevToolsAsync } from 'expo/devtools';
+
+const client = await connectPluginFromDevToolsAsync();
+
+client.sendMessage('ping', { from: 'app' });
+client.addMessageListener('pong', (data) => {
+  console.log('receiving pong', data);
+});
+```
+
+### Compatibility with Expo Go and Development builds
+
+Dev tools plugins should only include JavaScript code. They are generally compatible with Expo Go and [Development builds](/development-builds/introduction/) and should not require creating a new development build to add the plugin. If a package's underlying module that the plugin inspects includes native code and is not part of Expo Go, create a new development build to use both the component and the plugin from that package.
+
+For example, a dev tools plugin that inspects [React Native Firebase](/guides/using-firebase/#using-react-native-firebase) will not work with Expo Go. React Native Firebase includes native code that is not part of Expo Go. To use the dev tools plugin and React Native Firebase, create a development build.
+
+## Using a dev tools plugin
+
+After installing the dev tools plugin and adding the connecting required code to your project, you can start the dev server up with `npx expo start`. Then press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of available dev tools plugins. Select the plugin you want to use, and it will open in a new Chrome window.
+
+> When starting the dev server with the Expo CLI, there is an option to press <kbd>?</kbd> to **show all commands**. This shows additional commands, including the shortcut to open **more tools**. Dev tools plugins can also be selected in this menu.
+
+## Expo dev tools plugins
+
+Expo provides some dev tools plugins for common debugging tasks. Follow the instructions below to start using them in your app.
+
+### React Navigation
+
+Inspired by [`@react-navigation/devtools`](https://github.com/react-navigation/react-navigation/tree/4797ace/packages/devtools) and [`flipper-plugin-react-navigation`](https://github.com/react-navigation/react-navigation/tree/4797ace/packages/flipper-plugin-react-navigation), the React Navigation dev tools plugin allows seeing history of [React Navigation](https://reactnavigation.org/) actions and state. You can also rewind to previous points in your navigation history and send deep links to your app. Since Expo Router is built upon React Navigation, this plugin is fully compatible with [Expo Router](/router/introduction).
```

Comment:
```suggestion
Inspired by [`@react-navigation/devtools`](https://github.com/react-navigation/react-navigation/tree/main/packages/devtools) and [`flipper-plugin-react-navigation`](https://github.com/react-navigation/react-navigation/tree/main/packages/flipper-plugin-react-navigation), the React Navigation dev tools plugin allows seeing history of [React Navigation](https://reactnavigation.org/) actions and state. You can also rewind to previous points in your navigation history and send deep links to your app. Since Expo Router is built upon React Navigation, this plugin is fully compatible with [Expo Router](/router/introduction).
```

Let's use the `main` branch links instead of commit reference.